src/Headers/Tree.h

```diff
@@ -252,6 +257,13 @@ class Tree : public TreeBase<ndim>
     return BoxOverlap(ndim, box1min, box1max, box2min, box2max) ;
   }
 
+  virtual bool GetRelativeOpeningCriterion() const {
```

Comment:
Why is this virtual? Shouldn't use_relative_criterion be defined in the base class?